modules/testing-3-800.adoc

```diff
@@ -0,0 +1,82 @@
+[[testing-3-800]]
+== Testing 3.8.0 features 
+
+Use the following procedures to test how to enable new features, how to ensure they are working, and the expected results.
+
+=== Enabling and testing the IPv6 and dual-stack protocol family on standalone {productname} deployments 
+
+==== Enabling and testing IPv6 
+
+ IPv6 support is now available with Red Hat Quay 3.8. By enabling IPv6, your {productname} deployment better supports Telco and Edge environments, notably when {productname} serves as a central registry in the form of a hub cluster. 
+ 
+Use the following procedure to enable IPv6 on your standalone {productname} deployment. 
+
+.Prerequisites 
+
+* You have updated {productname} to 3.8.
+
+.Procedure 
+
+. In your deployment's `config.yaml` file, add the `FEATURE_LISTEN_IP_VERSION` parameter and set it to `IPv6`, for example: 
++
+[source.yaml]
+----
+---
+FEATURE_GOOGLE_LOGIN: false
+FEATURE_INVITE_ONLY_USER_CREATION: false
+FEATURE_LISTEN_IP_VERSION: IPv6
+FEATURE_MAILING: false
+FEATURE_NONSUPERUSER_TEAM_SYNCING_SETUP: false
+---
+----
+
+. Start, or restart, your {productname} deployment. 
+
+. Check that your deployment is listening to IPv6 by entering the following command:
++
+[source,terminal]
+----
+$ curl http://localhost:8080/health/instance
+{"data":{"services":{"auth":true,"database":true,"disk_space":true,"registry_gunicorn":true,"service_key":true,"web_gunicorn":true}},"status_code":200}
+----
++
+Alternatively, you can enter the following command to check if your deployment is listening to IPv6: 
++
+[source,terminal]
+----
+$ curl http://[::1]:8080/health/instance
+{"data":{"services":{"auth":true,"database":true,"disk_space":true,"registry_gunicorn":true,"service_key":true,"web_gunicorn":true}},"status_code":200}
+----
+
+==== Enabling and testing dual-stack  
+
+. In your deployment's `config.yaml` file, add the `FEATURE_LISTEN_IP_VERSION` parameter and set it to `IPv6`, for example: 
```

Comment:
what we want to say is 
```
In your deployment's `config.yaml` file, add the `FEATURE_LISTEN_IP_VERSION` parameter and set it to `dual-stack`, for example: 
```

Reply:
Thank you! same thing--just hadn't revised it yet (was copied from the earlier command). Sorry! Thanks!